fe/fe-core/src/main/java/org/apache/doris/catalog/ResourceMgr.java

```diff
@@ -52,7 +55,7 @@
     private static final Logger LOG = LogManager.getLogger(ResourceMgr.class);
 
     public static final ImmutableList<String> RESOURCE_PROC_NODE_TITLE_NAMES = new ImmutableList.Builder<String>()
-            .add("Name").add("ResourceType").add("Key").add("Value")
+            .add("NAME").add("RESOURCETYPE").add("KEY").add("VALUE")
```

Comment:
Why changing them to uppercase?
And as @xy720 said, better not using `KEY`, I prefer `Item`.